sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -185,6 +185,17 @@ final class DataFrameWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Specifies the name of the [[ContinuousQuery]] that can be started with `stream()`.
+   * This name must be unique among all the currently active queries in the associated SQLContext.
+   *
+   * @since 2.0.0
+   */
+  def queryName(queryName: String): DataFrameWriter = {
```

Comment:
Where should I put the test! There isnt any suite that tests the functionality of the DataFrameWriter.stream()